devpi_builder/cli.py

```diff
@@ -110,6 +116,11 @@ def main(args=None):
     parser.add_argument('--client-cert', help='Client key to use to authenticate with the devpi server.', default=None)
 
     args = parser.parse_args(args=args)
+    if args.user is None:
+        args.user = input('Username: ')
```

Comment:
`DEVPI_USER="" DEVPI_PASSWORD="" devpi-builder ..... `
will skip interactive prompt.
Let me know if you prefer add '--batch' or '--no-input'.